repo_data/distribution.xml.in

```diff
@@ -2778,5 +2778,8 @@
 
 		<!-- Replaced by eza -->
 		<Package>exa</Package>
+
+		<!-- Replaced by juliaup -->
+		<Package>julia</Package>
```

Comment:
You need to do _all_ of the packages built by the julia repo. From looking at the [unstable repo](https://packages.getsol.us/unstable/j/julia/) I see `julia`, `julia-devel`, `julia-docs`, and `julia-dbginfo`.

Reply:
Aw shoot made the same mistake I did with Haskell.